### PR TITLE
Making endpoint names constats to avoid errors

### PR DESCRIPTION
**Pull request by honza-kasik**
Please make sure your PR meets the following requirements:
- [x] Pull Request contains a description of the changes
- [x] Pull Request does not include fixes for multiple issues/topics
- [x] Code is formatted, imports ordered, code compiles and tests are passing
- [ ] N/A Link to the passing job is provided
- [x] Code is self-descriptive and/or documented
- [x] Description of the tests scenarios is included (see #46)

### CODE REVIEW

File: microprofile-jwt/src/test/java/org/jboss/eap/qe/microprofile/jwt/security/rbac/RolesAllowedRbacTest.java [hunk 1]
```diff
@@ -222,7 +223,7 @@ public void nonAuthenticatedAccessAdminDirectorPath(@ArquillianResource URL url)
      */
     @Test
     public void nonAuthenticatedAccessDenyAllPath(@ArquillianResource URL url) {
-        get(url.toExternalForm() + "rbac-endpoint/deny-all")
+        get(url.toExternalForm() + Endpoints.RBAC_ENDPOINT + "/deny-all")
```
Comment (istraka) [inline microprofile-jwt/src/test/java/org/jboss/eap/qe/microprofile/jwt/security/rbac/RolesAllowedRbacTest.java]: Any reason why not move `deny-all` to a constant?

File: microprofile-jwt/src/test/java/org/jboss/eap/qe/microprofile/jwt/security/rbac/RolesAllowedRbacTest.java [hunk 2]
```diff
@@ -166,7 +167,7 @@ public void monitorAccessDenyAllPath(@ArquillianResource URL url) {
 
         given().header("Authorization", "Bearer " + token.getRawValue())
                 .when()
-                .get(url.toExternalForm() + "rbac-endpoint/deny-all")
+                .get(url.toExternalForm() + Endpoints.RBAC_ENDPOINT + "/deny-all")
```
Comment (istraka) [inline microprofile-jwt/src/test/java/org/jboss/eap/qe/microprofile/jwt/security/rbac/RolesAllowedRbacTest.java]: Any reason why not move deny-all to a constant?

**Comment by istraka** [comment]
@honza-kasik did you run a job? Running TS locally is also good to me.

File: microprofile-jwt/src/test/java/org/jboss/eap/qe/microprofile/jwt/security/rbac/RolesAllowedRbacTest.java (hunk 1 above)
Comment (honza-kasik) [reply microprofile-jwt/src/test/java/org/jboss/eap/qe/microprofile/jwt/security/rbac/RolesAllowedRbacTest.java]: Good point! Thanks for finding this. Will update.

**Comment by honza-kasik** [comment]
@istraka I run the TS locally. This actually fixes few tests which I failed to refactor due to them being in parallel PR :/

**Comment by honza-kasik** [comment]
@istraka Updated, thanks!

**Pull request merged**